addons/viewdistance/XEH_postInit.sqf

```diff
@@ -5,17 +5,7 @@ if (!hasInterface) exitWith {};
 ["SettingsInitialized", {
     // if not enabled, then bugger off.
     if !(GVAR(enabled)) exitWith {};
-    
-    // Force the view distance down to the limit.
-    if (viewDistance > GVAR(limitViewDistance)) then {
-        setViewDistance GVAR(limitViewDistance);
-    };
-    
-    // Adapt view distance when the player is created or changed according to whether client is on foot or vehicle.
-    ["playerChanged",{
-        [false] call FUNC(adaptViewDistance);
-    }] call EFUNC(common,addEventHandler);
-    
```

Comment:
As it is you mean :grin: 

I'll try to hunt down @esteldunedain , if he won't answer in a few days (or someone else tests it) I'll readd it.